### ISSUE TITLE
Scoped Collection Assistance

### ISSUE THREAD

**Issue by ryanhmaas** [activeadmin/activeadmin#5404]
Hi all,

I'm having some difficulty scoping a collection/resource within a namespace. The hierarchy for my models looks something like:

`Invoice` -> `Property` -> `Property Manager` (where `->` denotes a `belongs_to` association). So in layman's terms, a property manager has many properties, which can have many invoices. 

I've tried things such as:

```
ActiveAdmin.register Invoice, namespace: :management do
    actions :index

    controller do
        def scoped_collection
            #ActiveRecord filtering using logged in user method.id here
        end
    end
end

```

but haven't had much luck. Thoughts on approach?

**Comment by yjukaku** [comment]
You must add `belongs_to` to your *ActiveAdmin* resource as well. That should solve any problems with filtering/scoping.
